site/public/js/drag-and-drop.js

```diff
@@ -369,21 +369,6 @@ function isValidSubmission(){
     return false;
 }
 
-function checkForPreviousSubmissions(csrf_token, gradeable_id, user_id){
```

Comment:
Just to confirm, this function isn't used anywhere and that's why you're deleting it?

Reply:
Yes. I did a full-text search and found no occurrences of `checkForPreviousSubmissions`.

Next time, if in a similar situation, is there a good way to tell reviewers that I delete something because it is not used?

Reply:
I think either by leaving an inline comment on the function (such as this), adding a comment about it in the first post of the PR, or make a separate PR that just removes this function.

Reply:
Sure. Thank you for your suggestion. Making a separate PR sounds good as it is easier to review. Deleting something from the codebase really makes me apprehensive considering how big the two files involved are.